conda_libmamba_solver/solver.py

```diff
@@ -921,7 +927,10 @@ def _check_spec_compat(self, match_spec: Union[MatchSpec, None]) -> Union[MatchS
                 f"You can only use {supported}, but you tried to use "
                 f"{tuple(unsupported_but_set)}.",
             )
-        if match_spec.get_raw_value("channel") == "defaults":
+        if (
+            match_spec.get_raw_value("channel") == "defaults"
```

Comment:
Yeah like @jezdez said nice catch. What caused you to find this issue?

Reply:
@chenghlee reported it :)